tools/prepare.py

```diff
@@ -131,8 +169,76 @@ def generate_txt(self):
         raise NotImplementedError
 
     # TODO add data visualize method, such that data can be checked every time after preprocess.
-    def visualize(self):
-        pass
+    def visualize(self, idx=None, alpha=0.8, mode="show"):
```

Comment:
what is the effect of the visualization? it is recommended to show it on the PR comment. 
This function can split into modules. Currently, it is a bit too complicate.